samples/python/prompt_lookup_decoding_lm/prompt_lookup_decoding_lm.py

```diff
@@ -5,11 +5,11 @@
 import argparse
 import openvino_genai
 
-def streamer(subword): 
-        print(subword, end='', flush=True) 
-        # Return flag corresponds whether generation should be stopped. 
-        # False means continue generation. 
-        return False
+def streamer(subword):
+    print(subword, end='', flush=True) 
+    # Return flag corresponds whether generation should be stopped. 
+    # False means continue generation. 
+    return False
```

Comment:
BTW, should we also support callback w/o return value?

E.g. when user don't care about any stop / cancellation